src/components/InfoPanel.component.js

```diff
@@ -98,6 +131,7 @@ class InfoPanel extends Component {
                   />
                 )}
               {displayContent}
+              <Carousel.Caption />
```

Comment:
you should be able to get rid of this since we're displaying the captions outside the carousel

Reply:
This I kept in because it creates a natural "barrier" at the bottom so the indicators will go underneath the image/video regardless of its size (if you take out Carousel.Caption, the indicators move up). The alternative would be setting a padding at the bottom purely through scss, which I feel might become subjective to the size of the screen (?)